jetty-start/src/main/java/org/eclipse/jetty/start/StartArgs.java

```diff
@@ -1505,6 +1505,12 @@ public void setProperty(String key, String value, String source)
             {
                 JavaVersion ver = JavaVersion.parse(value);
                 properties.setProperty("java.version.platform", Integer.toString(ver.getPlatform()), source);
+
+                // features built into java.
+                properties.setProperty("java.feature.alpn", Boolean.toString(ver.getPlatform() >= 9), source);
```

Comment:
Does "feature" add much?
How about just "java.alpn" ?

Reply:
If I saw `java.alpn` I would assume it's a alpn configuration, to tweak alpn behavior.

A different name I'm open to.
Perhaps we should probably make the prefix something that might not conflict in the future.
Stay away from the `java.` prefix?

Perhaps `jvm.feature.alpn` ?

Reply:
OK, I can live with "feature".    I think I have a slight preference for "jvm.feature" rather than "java.feature"

Reply:
But it's not a JVM feature - like a GC or JIT is.
It's really a Java API feature, so I think it should remain `java.feature.alpn`.

Reply:
Before I change anything here, I'm going to let this conversation simmer for a few days to see what develops.  See if opinions change (or not).

Reply:
I'm happy for you to leave the names as they are.